src/main/java/calculator/CalculatorService.java

```diff
@@ -0,0 +1,76 @@
+package calculator;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+
+public class CalculatorService {
+
+    public static void main(String[] args) {
+
+        Calculator calculator = new Calculator();
+        Scanner scan = new Scanner(System.in);
+
+        System.out.println("★ 사칙 연산 계산기 ★");
+        System.out.print("입력하시오 : ");
+
+        String calculatorText = scan.nextLine();
+
+        try {
+            calculateLoop(calculatorText, calculator);
+        } catch (IllegalArgumentException e) {
+            throw e;
+        }
+
+        System.out.print(calculatorText + " = ");
+        System.out.println(calculator.result);
+
+    }
+
+    static void calculateLoop(String text, Calculator calculator) {
+        List<String> list = Stream.of(text.split(" "))
+                .map(String::trim).collect(Collectors.toList());
+
+        for(int i=0; i < list.size(); i++) {
+            if(i==0) {
+                calculator.result = Long.parseLong(list.get(i));
+            }
+            else if(i%2==0) {
+                numberCheck(list.get(i));
+                calculate(calculator, list.get(i - 1), Long.parseLong(list.get(i)));
+            }
```

Comment:
스페이스도 컨벤션이 있습니다. 

```suggestion
        for (int i = 0; i < list.size(); i++) {
            if (i == 0) {
                calculator.result = Long.parseLong(list.get(i));
            } else if (i % 2 == 0) {
                numberCheck(list.get(i));
                calculate(calculator, list.get(i - 1), Long.parseLong(list.get(i)));
            }
        }
```

컨벤션을 지키지 위한 여러가지 방법이 있는데요. 찾아 보시고 적용해 보시면 좋을 것 같아요. 개인적으로는 손에 익숙해 지는 것이 가장 좋았습니다. :)